### DISCUSSION TITLE
How to use the "MRTD SOD Signer" ?

### DISCUSSION THREAD

**Discussion by HarimbolaSantatra** [Keyfactor/signserver-ce#95] [Q&A] [answered]
I'm currently experimenting different stuff with Sign Server.
How to use the [MRTD SOD Signer](https://doc.primekey.com/signserver/signserver-reference/signserver-workers/signserver-signers/mrtd-sod-signer) ?
On the client web, after putting the name of the worker on the "*name*" field, what should we put in the "*input*" field ?
I already tested different input but all yields the same 400 error:
```
Received request wasn't an expected SODSignRequest. 
```

I guess that this worker is only useful on the [signserver appliance](https://doc.primekey.com/signserver-appliance) with some sort of [smart card](https://doc.primekey.com/ejbca-appliance/operations/basic-hardware-operations/smart-card-handling) but I can be wrong!

Also, there's this section on the documentation:
> The MRTD SOD Signer creates the Complete Security Object (SOd) for Machine Readable Travel Documents (MRTD)

Is **Complete Security Object (SOd)** here the same as *Document Security Object* ?

Thanks!

**Comment by netmackan** [comment, marked as answer]
On the ClientWeb there is a sample form for uploading data groups to the MRTD SOD Signer.
Click on ClientWeb -> More... and the eMRTD page is displayed: https://localhost:8443/signserver/clientweb/mrtdsod.xhtml

> Is Complete Security Object (SOd) here the same as Document Security Object ?

Yes, that is the SOD from Doc 9303.

**Comment by HarimbolaSantatra** [comment]
Thanks you @netmackan !
